src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1055,7 +1055,7 @@ Do you agree to the terms?</value>
     <value>Moniker:</value>
   </data>
   <data name="ShowLabelPackageUrl" xml:space="preserve">
-    <value>Homepage:</value>
+    <value>Package Url:</value>

```

Comment:
I think that this one could be confusing, as some people may think that "package" means "installer".

Reply:
Apologies for the delay; Reverted